File: backend-seller-crud/src/main/java/com/example/backendsellercrud/model/Book.java
package com.example.backendsellercrud.model;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;


@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Book {

    @Id
    @GeneratedValue
    int id;

    String name;

    Integer price;

    private String language;
}

File: angular-crud/src/app/core/models/book.ts

export class Book {

  id: number;
   name: string;
   price: number;
   language: string;

  constructor(id: number, name: string, price: number, language: string) {
    this.id = id;
    this.name = name;
    this.price = price;
    this.language = language;
  }
}
